### PR TITLE
Ports & GAGS-ifies gas canisters from Yogstation

### PR DESCRIPTION
**Pull request by Wallemations**
## About The Pull Request
Ports gas canisters from Yogstation, with the consent of Partheos.
image
image

https://user-images.githubusercontent.com/66052067/152630143-02947c02-a1d3-4547-a8b0-11d7f0418c76.mp4

image

### I HAVE COME TO STEAL YOUR SOUL

Furthermore, the original canister sprites have long overstayed their welcome. They don't fit the 3/4ths top-down perspective, along with both scrubbers and those blue ones with fans on them. I saw that someone was working on updating those two, and thought that the canisters would look even more out of place if they weren't also updated.

## Changelog

:cl: Wallem, Partheo & Tex
imageadd: Ports canister sprites made by Partheo & Tex from Yogstation.
/:cl:

### CODE REVIEW

**Review by Imaginos16 — APPROVED** [review]
Mad lad

**Comment by Mothblocks** [comment]
it's a little hard to parse the wrenched canister sprites, im not even sure what the 1st and 3rd ones are

**Comment by LemonInTheDark** [comment]
image
RIP

**Comment by Imaginos16** [comment]
> it's a little hard to parse the wrenched canister sprites, im not even sure what the 1st and 3rd ones are

First shows the canister with air contents inside it, third shows it with none. At least, that was my understanding of it.

**Comment by Imaginos16** [comment]
> > it's a little hard to parse the wrenched canister sprites, im not even sure what the 1st and 3rd ones are
> 
> First shows the canister with air contents inside it, third shows it with none. At least, that was my understanding of it.

It also shows an oxygen tank being attached to it on the third one while wrenched, while the second one just shows it with a tank attached.

**Comment by Wallemations** [comment]
> it's a little hard to parse the wrenched canister sprites, im not even sure what the 1st and 3rd ones are

that's fair. I didn't know what to do about those. Maybe making it darker would help a little bit?
The second and third have tanks inside of them.
I'd love to figure out how to make the tank overlay change color depending on the tank inside, but as far as I know there's no way to check for what color a sprite majorly uses, and there's zero way I'd hard-code all that shit in lmao.

**Comment by Wallemations** [comment]
> it's a little hard to parse the wrenched canister sprites, im not even sure what the 1st and 3rd ones are
unknown
Hopefully this is a little more readable

**Comment by Wallemations** [comment]
image
They look MUCH better now

**Comment by Serxule** [comment]
they look much too grungy to really fit in tgs style

**Comment by Wallemations** [comment]
image

https://user-images.githubusercontent.com/66052067/152632845-83948d0d-b819-451e-905c-42a227984382.mp4

Here's a screenshot & video of all the canisters after the saturation boost & light change

**Comment by Serxule** [comment]
those do definitely look better, but the sprites themselves still look fine, (although it doesn't fit well with current tg style), they look sharp and jagged, primarily on the more detailed ones, maybe just a side effect of ss13 being pixels, but I don't notice that at all on the current sprites

**Comment by Bawhoppen** [comment]
Think some contrast could be good. 


>They look MUCH better now Two in the middle are fixed ones.

The ones on the outside I think are better visually
Otherwise the colors are too bright and you almost get a miniature version of snow-blindness from the sharp colors

**Comment by grimm-hollowknight** [comment]
its Partheo not Partheos

**Comment by Wallemations** [comment]
> its Partheo not Partheos

Fixed, sorry

**Comment by Dorsisdwarf** [comment]
the yellows are hard to parse

**Comment by SethLafuente** [comment]
the perspective and general sprite art dosent fit at all with the rest of the sprites in this game im not a fan of it

**Comment by SethLafuente** [comment]
Also could you give a better detailed say of why this is good for the game other than the joke is funny but dosent give any insight in why this is better than what we currently have

**Comment by Mothblocks** [comment]
i like them. theyll probably look a lot  better when connectors and stuff are 3/4 too

**Comment by Wallemations** [comment]
> Also could you give a better detailed say of why this is good for the game other than the joke is funny but dosent give any insight in why this is better than what we currently have 

Updated it, hope that helps

**Comment by Wallemations** [comment]
image
The lights on the canisters now will glow in the dark.

**Comment by cacogen** [comment]
Both cardinal sins of replacing sprites have been committed here

1. They look worse
2. They don't fit the art style

There's also the dual bonus of it being a port from another codebase (so not a situation where someone's effort will go to waste if the PR is closed) and a change to a longstanding sprite that didn't need it in the first place.

They kind of remind me of Fallout 1 and 2. I think because of the shading.

**Comment by notfrying1pans** [comment]
the lights on the canister on the right in this image don't look big enough
image

**Comment by Mothblocks** [comment]
"The art style" is what's being changed

**Comment by jacklecroy** [comment]
I don't think this fits well with the rest of the 3/4 sprites. It looks as if the camera angle is tilted much lower.

**Comment by Djaub** [comment]
Yeah it looks wrong, maybe the camera angle.

And, I don't know if it's a bug but the colour palette is weird: the canisters looks way too pale compared to their surrounding:
image
image
image
 
They don't really fit in...

**Comment by Bawhoppen** [comment]
I do think there's some color & shading issues personally
It's gonna be hard to fix that though cause of the GAGs

**Comment by cacogen** [comment]
>"The art style" is what's being changed

As usual, very insincere while defending a shitty thing. There's no consistency or deliberacy to the sprite changes. The different styles are just a side-effect of people changing random sprites to ones they prefer for no good reason (such as in this PR).

What's with the scare quotes around art style anyway? It's like you're  unwilling to acknowledge the basic reality that the game has a defacto art style reflected by the majority of its sprites. I'm not sure why you want to vandalise the game the way people do nowadays with their low quality and careless contributions by turning it into a mismatched mess of bad sprites that looks like multiple games and codebases rolled into one. 

As I've said in the past if the style of the game is somehow unacceptable now then come up with a new style to hold new sprites to. 

Otherwise hold new sprites to a basic standard of looking like the rest of the game. There are still sprite changes nowadays that do look like the game so clearly it's not an impossible standard to hold people to. Sprites have been being modernised without looking out of place since /tg/station began.

**Comment by Mothblocks** [comment]
The new art style is decided by the art maintainers (who merged the PR) and is one revolving around the 3/4 orientation

**Comment by drIcepop** [comment]
These just don't mix well with any other atmospherics sprite.

**Comment by ike709** [comment]
Regardless of any opinions on 3/4 perspective, because I think it can actually look nice when all of the sprites match the style, these canister sprites are simply ugly as sin and if that's what the art maintainers consider to be the new art style then you should reconsider who you have on the art maintainer team.

It looks like coder sprites for a bad Factorio mod.

**Comment by Bawhoppen** [comment]
The problem isn't the art style, it's this specific sprite has some significant problems. What bugs me is it's now gonna be a big pain to fix it due the the fact it uses GAGS, and that stuff is untouchable.

**Comment by optimumtact** [comment]
> The problem isn't the art style, it's this specific sprite has some significant problems. What bugs me is it's now gonna be a big pain to fix it due the the fact it uses GAGS, and that stuff is untouchable.

issue is one of skill?
